src/main/java/xyz/groundx/caver_ext_kas/rest_client/io/swagger/client/api/wallet/model/AccountByPubkey.java

```diff
@@ -42,7 +42,7 @@ public AccountByPubkey address(String address) {
    * Klaytn account address
    * @return address
   **/
-  @Schema(example = "0x0b7caaf70e7A0a5399041c64711E535CE8B3cf7d", required = true, description = "Klaytn account address")
+  @Schema(example = "65579084041275922354965185915039153946348474237", required = true, description = "Klaytn account address")
```

Comment:
It looks like it's not the address value. Is this value is valid?

Reply:
This example data reproduced by swagger-codegen by using Wallet API open api spec.